assets/scss/base/_table.scss

```diff
@@ -120,3 +120,7 @@ table {
     float: left;
   }
 }
+
+.table--font-small {
```

Comment:
Thanks for that dude.

The text sits inside a TD tag, not a P. Why do you suggest `table--reset`? It doesn't really tell a user what the selector does. (By the way, the selector goes on the TD, not the TABLE element.)